frontend/src/components/SortableTaskList.tsx

```diff
@@ -11,8 +11,10 @@ export const SortableTaskList: FC<{
   listId: string;
   tasks: Task[];
   disableDragging: boolean;
-}> = ({ listId, tasks, disableDragging }) => {
-  return (
+  showRatings?: boolean;
```

Comment:
Boolean is nicer if you need to use a variable or expression to specify the prop, otherwise the use would need something like

```
showRatings={boolExpr || undefined}
```

Reply:
True if we need to use it as such in the future 👍 I only questioned this since currently showRatings does not depend on variables anywhere